### PR TITLE
Add a mod to remove railroad stations

### PR DESCRIPTION
**Pull request by Inglonias**
#### Summary
```SUMMARY: Mods "Adds a mod to remove above ground railroad stations"```
#### Purpose of change
I personally do not like that railroad stations are around, but not done yet. Feels weird.

#### Describe the solution
Not everyone would agree with me, and I don't want to fight them on that. I made a mod, and I want to distribute it.

#### Describe alternatives you've considered
I dunno, putting this thing on [NexusMods](https://www.nexusmods.com/cataclysmdarkdaysahead) instead? If you want me to do that, let me know.

#### Additional context
N/A

### CODE REVIEW

File: data/mods/No_Rail_Stations/modinfo.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[
+  {
+    "type": "MOD_INFO",
+    "ident": "No_Rail_Stations",
+    "name": "No Rail Stations",
+    "authors": [ "Inglonias" ],
+    "description": "Removes above-ground rail stations from the game.",
+    "category": "monster_exclude",
```
Comment (Inglonias) [inline data/mods/No_Rail_Stations/modinfo.json]: I think this is the wrong category for this mod. Anybody got a better one?

Comment (ZhilkinSerg) [reply data/mods/No_Rail_Stations/modinfo.json]: Buildings

Comment (Inglonias) [reply data/mods/No_Rail_Stations/modinfo.json]: That puts it into the first column of mods, rather than the exclusion section, but fine. Done.

Comment (ZhilkinSerg) [reply data/mods/No_Rail_Stations/modinfo.json]: That mod selection menu needs rework anyway.

**Comment by paulenka-aleh** [comment]
I'd say railroads (as unfinished stuff) deserve to be in a mod themselves and shouln't be on by default since stations without actual railroads do look weird

**Comment by Night-Pryanik** [comment]
Not necessarily. We had lots of stuff in the past that was added incomplete and then got improved gradually.

**Comment by Rail-Runner** [comment]
Isn't that the reason there is a division between stable and experimental versions? People who prefer stability and finished content could just get a stable version.

**Comment by l29ah** [comment]
I think those weird railroad stations are present in 0.D release.

**Comment by Inglonias** [comment]
Indeed they are. Pretty sure this mod (or a modified version) would work for that version too. Regardless, pushing to merge into the 0.D branch is a bad plan, so...

**Comment by Inglonias** [comment]
Jenkins rebuild

**Comment by Inglonias** [comment]
This is ready to be merged as far as I can tell. Gorgon is being strange and not actually running its tests on this branch. No idea why.

**Comment by Inglonias** [comment]
Jenkins rebuild

**Comment by JazzGuru** [comment]
Glad to see this.  It always annoyed me when one of these would pop up in my starting area, without the track actually connecting to anything else.

Until railroads are done properly, it will definitely be staying in my mod-list... :)

**Pull request merged**